keps/sig-instrumentation/2305-metrics-cardinality-enforcement/README.md

```diff
@@ -266,14 +270,43 @@ This would then be interpreted by our machinery as this:
 ```
 
 ## Design Details
+
 ### Test Plan
-For `Alpha`, unit test to verify that the metric label will be set to "unexpected" if the metric encounters label values outside our explicit allowlist of values.
+
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+N/A
+
+##### Unit tests
+
+For `Alpha`, unit test to .
+
+- `staging/src/k8s.io/component-base/metrics/counter_test.go`: `3/3/2021` - `verify that the metric label will be set to "unexpected" for counters if the metric encounters label values outside our explicit allowlist of values`
+- `staging/src/k8s.io/component-base/metrics/gauge_test.go`: `4/3/21` - `verify that the metric label will be set to "unexpected" for gauges if the metric encounters label values outside our explicit allowlist of values`
+- `staging/src/k8s.io/component-base/metrics/histogram_test.go`: `4/3/21` - `verify that the metric label will be set to "unexpected" for histograms if the metric encounters label values outside our explicit allowlist of values`
+- `staging/src/k8s.io/component-base/metrics/summary_test.go`: `4/3/21` - `verify that the metric label will be set to "unexpected" for summaries if the metric encounters label values outside our explicit allowlist of values`
+
 ### Graduation Criteria
-For `Alpha`, the allowlist of metrics can be configured via the exposed flag and the unit test is passed.
-For `Beta`, the allowlist can be configured from a input file(e.g. yaml file).
+
+#### Alpha
+
+- Feature implemented behind a feature flag
+- The allowlist of metrics can be configured via the exposed flag and the unit test is passed.
+
+#### Beta
+
+- The allowlist can be configured from a manifest.
+
 ### Upgrade / Downgrade strategy
+
 N/A
+
 ### Version Skew Strategy
+
 N/A
 
 ## Production Readiness Review Questionnaire
```

Comment:
> What happens if we reenable the feature if it was previously rolled back?

It would be good to note metrics are a memory-only construct and do not persist across restarts.